component.html

```diff
@@ -0,0 +1,152 @@
+          </a>
+        </li>
+        <li class="navbar-list-item">
+          <a href="https://twitter.com/agarwal_yashoda?s=11">
+            <i class="fab fa-twitter icons"></i>
+          </a>
+        </li>
+      </ul>
+    </nav>
+
+    <!-- ---- sidebar ---- -->
+    <div class="sidebar">
+      <h3>Components</h3>
+      <ul class="nav">
+        <li>
+          <a href="#avatars">
+            <span>Avatar</span>
+          </a>
+        </li>
+        <li>
+          <a href="#alerts">
+            <span>Alert</span>
+          </a>
+        </li>
+        <li>
+          <a href="#badges">
+            <span>Badge</span>
+          </a>
+        </li>
+        <li>
+          <a href="buttons">
+            <span>Button</span>
+          </a>
+        </li>
+        <li>
+          <a href="#cards">
+            <span>Card</span>
+          </a>
+        </li>
+        <li>
+          <a href="#images">
+            <span>Image</span>
+          </a>
+        </li>
+        <li>
+          <a href="#Inputs">
+            <span>Input</span>
+          </a>
+        </li>
+      </ul>
+    </div>
+
+    <!-- ----- avatar ----- -->
+
+    <section class="main-title" id="avatars">
+      <h3 class="avatar-heading">Avatar</h3>
+      <p class="component-navbar-item">
+        In computing, an avatar (also known as a profile picture or userpic) is
+        a graphical representation of a user or the user's character or persona.
+      </p>
+      <div class="avatar-items">
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-xl" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-l" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-md" />
+        <img src="./Asset/kitty.png" alt="" class="kitty avatar-s" />
+      </div>
+
+      <iframe
+        src="https://carbon.now.sh/embed?bg=rgba%28248%2C192%2C200%2C0.94%29&t=base16-light&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=46px&ph=49px&ln=false&fl=1&fm=Droid+Sans+Mono&fs=13.5px&lh=130%25&si=false&es=2x&wm=false&code=%250A%2520%2520%253Cimg%2520src%253D%2522%2522%2520alt%253D%2522%2522%2520class%253D%2522kitty%2520%2520avatar-xl%2522%2520%252F%253E%250A%2520%2520%253Cimg%2520src%253D%2522%2522%2520alt%253D%2522%2522%2520class%253D%2522kitty%2520%2520avatar-l%2522%2520%252F%253E%250A%2520%2520%253Cimg%2520src%253D%2522%2522%2520alt%253D%2522%2522%2520class%253D%2522kitty%2520%2520avatar-md%2522%2520%252F%253E%250A%2520%2520%253Cimg%2520src%253D%2522%2522%2520alt%253D%2522%2522%2520class%253D%2522kitty%2520%2520avatar-s%2522%2520%252F%253E%250A"
+        style="
+          width: 100%;
+          height: 254px;
+          border: 0;
+          transform: scale(1);
+          overflow: hidden;
+        "
+        sandbox="allow-scripts allow-same-origin"
+      >
+      </iframe>
+    </section>
+
+    <!-- ---- alert ---- -->
+
+    <section class="main-title" id="alerts">
```

Comment:
classes will be enough to use.
use ID when you want to override some properties

Reply:
I have used this id to connect it to my sidebar

Reply:
ok. cool💯